src/app/helpers/backbone/view.js

```diff
@@ -224,7 +245,15 @@ class BaseView extends Backbone.View {
 
     onShow() {} // noop
     onBeforeRender() {} // noop
-    onRender() {} // noop
+
+    onRender() {
+        this.attachListenerTo(window, 'scroll', () => {
+            if (this.freezePosition) {
+                window.scrollTo(0, this.freezePosition);
+            }
+        });
```

Comment:
This particular bit is for when you're scrolling a popover, and you reach the end of its range, the page starts scrolling.